Makefile

```diff
@@ -18,19 +24,68 @@ lint:
 	@echo Running gocilint...
 	@golangci-lint run --out-format=tab --issues-exit-code=0
 
+help: Makefile
```

Comment:
Show
```
 Choose a command run in , or just run 'make' for install

  govet       Run go vet.
  format      Run gofmt and goimports.
  lint        Run Golang CI Lint.
  test-unit   Run the unit tests.
  test-unit   Run the unit tests checking for race conditions
  test-unit   Run the unit tests and create a coverate html report
  test
```
`test-unit` instead of `test-race` and `test-cover`